pkg/helper/yaml.go

```diff
@@ -0,0 +1,19 @@
+package helper
```

Comment:
This `helper` package I did was a very bad at golang we create new packages for stuff like this so can we do a pkg/helper/render/yaml.go ? similar to CNAO

Reply:
Fixed, I think, please check I got you